tdrs-backend/tdpservice/email/email.py

```diff
@@ -61,7 +60,7 @@ def prepare_email(email_path, recipient_email, email_context, logger_context):
 
     return html_message, valid_emails
 
-@shared_task
```

Comment:
Just curious why this isn't a celery task anymore?

Reply:
For some reason the description wasnt updating for me. I see the answer to my question in it now ha.